tests.edn

```diff
@@ -1,17 +1,15 @@
 #kaocha/v1
-        #meta-merge [{:tests    [{:id          :clj
+        #meta-merge [{:tests    [{:id          :clj-pss
                                   :focus-meta [:focused]
```

Comment:
What is this supposed to do? 
I would think that this selects test functions with the selected metadata flags, but I cannot find any functions like that.

Reply:
seems like a leftover 🤷

Reply:
same metadata-flag as below